website/source/docs/agent/caching/index.html.md

```diff
@@ -145,6 +194,40 @@ These configuration values are common to all Listeners.
 - `tls_cert_file` `(string: optional)` - Specifies the path to the certificate
   for TLS.
 
+### Configuration (`vault`)
+
+There can at most be one top level `vault` block and it has the following
+configuration entries:
+
+- `address (string: optional)` - The address of the Vault server. This should
```

Comment:
We should mention that 127.0.0.1:8100 is the default value.

Reply:
This is not the agent's address. This is the Vault server's address, and there is no default.

Reply:
Oh right